app/views/capacity.html

```diff
@@ -40,6 +40,71 @@
 
 <div id="plotly-div-line" role="presentation"></div>
 
+{% if organisationLevel !== 'offender-manager' %}
+  <h3 class="heading-medium">Capacity breakdown</h3>
+  <table id="example" class="data-table dataTable">
+    <thead>
+      <tr>
+        <th colspan="2" id="team">{{ childOrganisationLevelDisplayText }}</th>
+        <th class="" colspan="14" id="breakdown">Capacity breakdown</th>
+      </tr>
+      <tr>
+        <th id="name">Name</th>
+        <th id="grade">Grade</th>
+        <th id="capacity">Capacity(%)</th>
+        <th id="cases">Cases</th>
+        <th id="paroms"><abbr title="Parole assessment reports by the Offender Manager">PAROMs</abbr></th>
+        <th id="fdrs"><abbr title="Fast delivery reports">FDRs</abbr></th>
+        <th id="sdrs"><abbr title="Standard delivery reports">SDRs</abbr></th>
+        <th id="arms"><abbr title="Active Risk Management System">ARMS</abbr></th>
+        <th id="gs"><abbr title="Group supervision">GS</abbr></th>
+        <th id="cms"><abbr title="Case management support">CMS</abbr></th>
+      </tr>
+    </thead>
+    {% if organisationLevel === 'team' %}
+      <tbody>
+        {% for member in capacityBreakdown %}
+        <tr>
+          <td headers="name team"><a href="/offender-manager/{{ member.linkId }}">{{ member.name }}</a></td>
```

Comment:
If the above change was implemented `team` -> `child` then these references should also be updated.

Reply:
updated